src/components/PostList.jsx

```diff
@@ -7,13 +7,26 @@ const PostList = ({
   handleLikeCount,
   openCommentBox,
   postList,
+  onSubmitComment,
+  onDeleteComment,
 }) => {
   const { name } = useContext(UserContext);
+  const [writeComment, setWriteComment] = useState("");
+
+  const submitComment = (id) => {
+    writeComment && onSubmitComment(writeComment, id);
+    setWriteComment("");
+  };
+
+  const handleDeleteComment = (postId, commentId) => {
+    onDeleteComment(postId, commentId);
+  };
 
   return (
     <div className="post-collection">
       {postList?.map((post) => {
```

Comment:
postList should always be empty array. So no need to set it to `null` or `undefined`.
```suggestion
      {postList.map((post) => {
```